book/src/bridge/extern_rustqt.md

```diff
@@ -102,27 +108,22 @@ The `#[qproperty(TYPE, NAME, ...)]` attribute can be specified on a [`#[qobject]
 {{#include ../../../examples/qml_minimal/rust/src/cxxqt_object.rs:book_rustobj_struct_signature}}
 ```
 
-The type and name of the
+If no other attributes are specified on the property, CXX-Qt will generate setters and getters, as well as a "changed" signal automatically.
+The type and name of the property must then match a field in the inner Rust struct.
 
 ```rust,ignore,noplayground
 {{#include ../../../examples/qml_minimal/rust/src/cxxqt_object.rs:book_rustobj_struct}}
 ```
 
-For every `#[qproperty]`, CXX-Qt will generate setters and getters, as well as a "changed" signal.
-
-On the C++ side:
-
-- setter: `set<Property>`
-- getter: `get<Property>`
-- changed: `<Property>Changed`
+CXX-Qt will then generate these functions:
 
-On the Rust side:
+|                | C++                | Rust                 |
+|----------------|--------------------|----------------------|
+| setter         | `set<Property>`    | `set_<Property>`     |
+| getter         | `get<Property>`    | `<Property>`         |
+| changed signal | `<Property>Changed`| `<Property>_changed` |
```

Comment:
do we need to now how in C++ for set/get the first letter becomes upper case?

eg `set<MyProperty>` and then `set_<myProperty>` in Rust etc

Reply:
Added a footnote and used `<property>` and `<Property>` respectively.